docs/metrics.md

```diff
@@ -0,0 +1,171 @@
+# METRICS
+
+## Data Analysis
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do users install and run this?
+
+What is the overall engagement of the Containers experiment?
+**This is the standard Daily Active User (DAU) and Monthly Active User (MAU) analysis.**
+
+This captures data from the users who have the add-on installed, regardless of
+whether they are actively interacting with it.
+
+![](images/kpi-1.png)
+
+### Immediate Questions
```

Comment:
Based on @TanviHacks's comments no - we're not going to measure default container usage.